base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -165,7 +165,7 @@ impl fmt::Display for OutputManagerRequest {
                 "Encumber aggregate utxo with tx_id: {} and output: ({},{})",
                 tx_id,
                 expected_commitment.to_hex(),
-                output_hash
+                output_hash.to_hex()
```

Comment:
```suggestion
                output_hash
```

HashOutput already implements to_hex for display